scripts/gen_vpc_ip_limits.go

```diff
@@ -114,6 +114,9 @@ func main() {
 		Timestamp: time.Now().Format(time.RFC3339),
 		ENILimits: eniLimits,
 	})
+	if err != nil {
+		log.Fatalf("Failed to generate template: %v\n", err)
```

Comment:
Is it ok not to return if it failed to generate the template?

Reply:
It does exit here, [`log.Fatalf` calls `os.Exit(1)` after printing the error message](https://golang.org/pkg/log/#Fatalf).

Reply:
Ahh ok, missed it. Thanks.